server/routes/static.py

```diff
@@ -28,6 +28,8 @@ def homepage():
     return render_template('custom_dc/iitm/homepage.html')
   if g.env_name == 'STANFORD':
     return render_template('custom_dc/stanford/homepage.html')
+  if g.env_name == 'STANFORD-STAGING':
+    return render_template('custom_dc/stanford/homepage.html')
```

Comment:
nit: this shouldn't be needed also..

Reply:
I thought about this. Why not? This isn't for local development, right? So don't we have to have a way to differentiate, even though right now they serve the same page?

Reply:
well, mostly because we no longer have an env_name = STANFORD-STAGING. also, we would want all "stanford" instances, local or otherwise, to exhibit the same behavior.